.storybook/stories/LendCtaExp.stories.js

```diff
@@ -49,49 +38,52 @@ const queryFullyFunded = {
 export default {
     title: 'Components/Lend CTA Exp',
     component: LendCtaExp,
-    args: { loanId: loan.id },
+    args: { loan: loan },
 };
 
 export const Default = (_args, { argTypes }) => ({
     props: Object.keys(argTypes),
 	components: { LendCtaExp },
-	mixins: [apolloStoryMixin({ queryResult }), cookieStoreStoryMixin()],
```

Comment:
The mixin imports can be removed from this file.

Reply:
Fixed